server/server_test.go

```diff
@@ -886,14 +883,14 @@ func (cli *testServerClient) runTestLoadData(t *testing.T, server *Server) {
 		dbt.MustExec("create sequence s1")
 
 		// can't insert into views (in TiDB) or sequences. issue #20880
-		_, err = dbt.GetDB().Exec("load data local infile '/tmp/load_data_test.csv' into table v1")
+		_, err = dbt.GetDB().Exec(fmt.Sprintf("load data local infile %q into table v1", path))
```

Comment:
```
_, err := dbt.GetDB().Exec("load data local infile ? into table t", path)
```

Reply:
No, it doesn't support this protocol:

```
    server_test.go:889:
                Error Trace:    server_test.go:889
                                                        server_test.go:151
                                                        server_test.go:877
                                                        tidb_serial_test.go:45
                Error:          Not equal:
                                expected: "Error 1105: can only load data into base tables"
                                actual  : "Error 1064: You have an error in your SQL syntax; check the manual that corresponds to your TiDB version for the right syntax to use line 1 column 24 near \"%? into table v1\" "

                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -1 +1 @@
                                -Error 1105: can only load data into base tables
                                +Error 1064: You have an error in your SQL syntax; check the manual that corresponds to your TiDB version for the right syntax to use line 1 column 24 near "%? into table v1"
                Test:           TestLoadData1
FAIL
```

You can try this SQL:

```
mysql> prepare stmt from 'load data local infile ? into table v1';
ERROR 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your TiDB version for the right syntax to use line 1 column 24 near "? into table v1"
```

@hawkingrei